composer/utils/checkpoint.py

```diff
@@ -202,7 +264,7 @@ def _download_checkpoint(
             # every NODE needs the GLOBAL rank zero checkpoint
             path = _format_path_with_rank_zero(path)
             get_file(destination=rank_zero_checkpoint_filepath,
-                     path=path,
+                     path=_follow_symlink(_format_path_with_rank_zero(path), node_checkpoint_folder, 0),
```

Comment:
Previously, this didn't have `_format_path_with_rank_zero`, so it seems like it would have used a path which still had `{rank}` in it. It looked like a bug to me so I've changed it to format -- would be great if someone who is more familiar with this code can confirm